gnssanalysis/gn_download.py

```diff
@@ -865,26 +833,30 @@ def download_product_from_cddis(
                 remain = end_epoch - reference_start
 
 
-def download_atx(download_dir: _Path, long_filename: bool = False, if_file_present: str = "prompt_user") -> None:
-    """
-    Download the ATX file necessary for running the PEA provided the download directory (download_dir)
-    """
+def download_atx(download_dir: _Path, long_filename: bool = False, if_file_present: str = "prompt_user") -> _Path:
+    """Download the ATX file necessary for running the PEA provided the download directory (download_dir)
 
+    :param _Path download_dir: Where to download files (local directory)
+    :param bool long_filename: Download ATX file relevant after 2022-11-27? True: igs20, False:igs14, defaults to False
```

Comment:
```suggestion
    :param bool long_filename: Indicates whether to download the newer ATX file (igs20), typically used with long format filenames (post 2022-11-27), or use the older igs14. Defaults to False (igs14).
```

I'm not sure if I've misinterpreted what this was saying, but suffice it to say the meaning wasn't clear. 😅

Reply:
I've changed the name of the argument to make it clearer. We will need to make a subsequent change in the ginan repo for `auto_download_PPP`